docs/en/ufs/HDFS.md

```diff
@@ -80,6 +80,12 @@ $ bin/alluxio format
 $ bin/alluxio-start.sh local
 ```
 
+If your ramdisk is not mounted, likely because this is the first time you are running Alluxio, you may need to start Alluxio with the `SudoMount` option.
+
+```bash
+$ bin/alluxio-start.sh local SudoMount
+```
```

Comment:
Include both with and without `SudoMount` so they don't get confused about what to do if they are not running for the first time (ie. they might skip the command altogether).

Reply:
the command above this shows without `SudoMount` option.